.github/workflows/run-archiver-cambium.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: run-archiver-cambium
+
+on:
+  workflow_dispatch:
+    inputs:
+      datasets:
+        description: 'Comma-separated list of datasets to archive (e.g., `"nrelcambium2020","nrelcambium2023"`).'
+        default: ""
+        required: true
+        type: string
+      create_github_issue:
+        description: "Create a Github issue from this run?"
+        default: false
+        required: true
+        type: boolean
+
+jobs:
+  archive-run:
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        # Note that we can't pass global env variables to the matrix, so we manually reproduce the list of datasets here.
+        dataset: ${{ fromJSON(format('[{0}]', inputs.datasets || '"nrelcambium2020","nrelcambium2021","nrelcambium2022","nrelcambium2023"' )) }}
+      fail-fast: false
+    runs-on: ubuntu-latest-4core
```

Comment:
In theory we could set this `runs-on` string via the workflow input. That would let us use the normal `run-archiver` action to manually trigger any datasets on any size runners, which seems like it could be useful in the future.

In practice, it looks like it doesn't quite work, and requires a [janky workaround.](https://stackoverflow.com/questions/72633058/parameterize-runs-on-value-for-github-action)

Your current solution works now, makes it really obvious how to run archivers on big runners, and already lets us run any datasets we want on said big runners. So I think we should just ship this, though maybe we should rename this to `run-archiver-large`. We can add a little TODO on either workflow that pushes us to attempt reunification the next time we touch the workflows.